stock-plugins/aiverify.stock.image-corruption-toolbox/algorithms/blur_corruptions/aiverify_blur_corruptions/algo.py

```diff
@@ -141,20 +141,21 @@ def __init__(
         self._results = {"results": [0]}
         self._ordered_ground_truth = None
         self._tmp_path = self._base_path / "temp"
```

Comment:
No need to output the files in `_tmp_path`. Instead since we now have an `output` folder, let's include the generated files in the output folder. This will then be specified as [artifacts](https://github.com/aiverify-foundation/aiverify/blob/v2.x/common/schemas/aiverify.testresult.schema.json#L106) for the plugin

Reply:
@timlrx  : Have updated the path, so all temp files (temp/, widgets/ ) will be written to output folder.
Also the path in display_info is now updated to algorithms/blur_corruptions/output/temp/Gaussian_Blur/severity0/9.png

(see screenshot below)

<img width="1336" alt="image" src="https://github.com/user-attachments/assets/f0941a2d-0c14-4337-8812-2e77c2780bd0">